<launch>
	<node name="PatternGenXY" pkg="patterngen" type="PatternGenXY.py" /> 
</launch>

